speechbrain/lobes/models/transformer/Transformer.py

```diff
@@ -420,13 +422,16 @@ def __init__(
             ]
         )
         self.norm = sb.nnet.normalization.LayerNorm(d_model, eps=1e-6)
+        self.layerdrop_prob = layerdrop_prob
+        self.rng = np.random.default_rng(0)  # for DDP so same layer dropped
```

Comment:
Replicability here: Should this be set with the seed of the yaml instead ? Always bad to have some constant in the code.